legal-api/src/legal_api/models/business.py

```diff
@@ -94,6 +95,20 @@ class LegalTypes(Enum):
         MISC_FIRM = 'MF'
         FINANCIAL = 'FI'
         CONT_IN_SOCIETY = 'CS'
+        # UNLIMITED_LIABILITY_COMPANY = 'UL'
+        # SOLE_PROPRIETORSHIP = 'FR'
+        # GENERAL_PARTNERSHIP = 'GP'
```

Comment:
a bunch of these are duplicates of ones already in the enum, according to the code and types.
these should be removed

Reply:
Good catch. I merged the duplicate enums.